google-cloud-logging/src/test/java/com/google/cloud/logging/LoggingImplTest.java

```diff
@@ -2183,4 +2226,41 @@ public void run() {
     }
     assertSame(0, exceptions.get());
   }
+
+  private void test_write_log_entries_with_destination(
+      String projectId, String logName, LogDestinationName destination) {
+    Map<String, String> labels = ImmutableMap.of("key", "value");
+    WriteLogEntriesRequest request =
+        WriteLogEntriesRequest.newBuilder()
+            .putAllLabels(labels)
+            .setLogName(logName)
+            .setResource(MONITORED_RESOURCE.toPb())
+            .addAllEntries(
+                Iterables.transform(
+                    ImmutableList.of(
+                        LOG_ENTRY1,
+                        LOG_ENTRY_BILLING,
+                        LOG_ENTRY_FOLDER,
+                        LOG_ENTRY_ORGANIZATION,
+                        LOG_ENTRY_NO_DESTINATION,
+                        LOG_ENTRY_EMPTY),
+                    LogEntry.toPbFunction(projectId)))
+            .build();
+    WriteLogEntriesResponse response = WriteLogEntriesResponse.newBuilder().build();
+    EasyMock.expect(loggingRpcMock.write(request)).andReturn(ApiFutures.immediateFuture(response));
+    EasyMock.replay(rpcFactoryMock, loggingRpcMock);
+    logging = options.getService();
+    logging.write(
+        ImmutableList.of(
+            LOG_ENTRY1,
+            LOG_ENTRY_BILLING,
+            LOG_ENTRY_FOLDER,
+            LOG_ENTRY_ORGANIZATION,
+            LOG_ENTRY2,
+            LOG_ENTRY_EMPTY),
+        WriteOption.logName(LOG_NAME),
+        WriteOption.resource(MONITORED_RESOURCE),
+        WriteOption.labels(labels),
+        WriteOption.destination(destination));
+  }
```

Comment:
what does this test assert? it looks like there is no comparison between expected and actual results.
i recommend to rewrite the test in a way that it includes validation that given the input the expected results match the actual results.

Reply:
It does validates - I believe this happens in `tearDown()`

Reply:
I see. Indeed, the line 2250 should do the trick, but it is very hard to spot.
nit: Would you mind to give to the `request` variable a more "descriptive" name? Something like `expectedWriteLogEntriesRequest` or similar. It will let to a reader to better understand what we assert in the call to `expect()`.

Reply:
Sure